apps/nextjs-website/src/components/atoms/TabComponent/TabComponent.tsx

```diff
@@ -6,37 +6,61 @@ type TabPanelProps = {
   readonly children?: ReactNode;
   readonly index: number;
   readonly value: number;
+  readonly px?: number;
```

Comment:
maybe we could pass a style prop to have the possiblity to further modify this later?